openapi/components/schemas/assessment-item.yaml

```diff
@@ -50,10 +51,14 @@ properties:
               description: The name of the year level.
   commonAssessment:
     type: boolean
+    nullable: true
+    description: Null if assessmentType is 'lessonPlan'.
```

Comment:
I am pretty sure more than just lesson plans will have null for common assessment...  In fact the only object that can have a commonAssessment flag is "Due Work"